.github/workflows/checks.yml

```diff
@@ -0,0 +1,74 @@
+name: Tests
+on: [push]
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dmenv
+        uses: TankerHQ/dmenv-install-action@v1.0.0
+        with:
+          dmenv-version: 0.20.0
+      - name: Prepare project for development
+        run: dmenv install
+      - name: Run tests
+        run: dmenv run -- pytest
+
+  python34:
```

Comment:
why is python3.4 special?

Reply:
It's not avaible on the latest ubuntu: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts

Reply:
Fair enough